sccm/sum/deploy-use/automatically-deploy-software-updates.md

```diff
@@ -229,10 +229,10 @@ After you create an ADR, add additional deployments to the rule. This action hel
      - Alerts
      - Download Settings  
 
+Deployments can also be added programmatically using powerShell cmdelets. See [New-CMSoftwareUpdateDeployment](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/new-cmsoftwareupdatedeployment) for a complete description of using this method.

```

Comment:
Some minor issues to fix before publishing:
1. Change `powerShell` to `Windows PowerShell`
2. Change `cmdelets` to `cmdlets`
3. Remove `en-us/` from the URL to make it language neutral
4. Restructure the second sentence using the standard style: `For a complete description of using this method, see [New-CMSoftwareUpdateDeployment](https://docs.microsoft.com/powershell/module/configurationmanager/new-cmsoftwareupdatedeployment).`